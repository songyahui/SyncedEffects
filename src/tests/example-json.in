;
;
TIN({"hours":4, "minutes":5, "seconds":6, "ampm": true});
;
;

