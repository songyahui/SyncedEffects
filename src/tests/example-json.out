example-json> ;
--- Output: Time({"hours":0,"minutes":0,"seconds":1,"ampm":false})
example-json> ;
--- Output:
example-json> TIN({"hours":4, "minutes":5, "seconds":6, "ampm": true});
--- Output: Time({"hours":4,"minutes":5,"seconds":6,"ampm":true})
example-json> ;
--- Output: Time({"hours":4,"minutes":5,"seconds":7,"ampm":true})
example-json> ;
--- Output:
